src/App.jsx

```diff
@@ -13,6 +15,8 @@ function App() {
         <Route path="/" element={<Home />} />
         <Route path="/reviews" element={<Reviews />} />
         <Route path="/reviews/:review_id" element={<SingleReview />} />
+        <Route path="/categories" element={<Categories />} />
+        <Route path="/categories/:category" element={<SingleCategory />} />
```

Comment:
This `SingleCategory` component is doing exactly what your `Reviews` component is doing. Basically, they both render a list of reviews. I understand that Single Category is also filtering by category, however, this can also be achieved in the Reviews component by making a slight change to your useEffect